doc/source/visualization.rst

```diff
@@ -456,28 +456,39 @@ columns:
 
 .. _visualization.box.return:
 
-Basically, plot functions return :class:`matplotlib Axes <matplotlib.axes.Axes>` as a return value.
-In ``boxplot``, the return type can be changed by argument ``return_type``, and whether the subplots is enabled (``subplots=True`` in ``plot`` or ``by`` is specified in ``boxplot``).
+.. warning::
+
+   The default changed from ``'dict'`` to ``'axes'`` in version 0.19.0.
+
+Most plot functions return a scalar or an array of :class:`matplotlib Axes <matplotlib.axes.Axes>`.
+In ``boxplot``, the return type can be controlled by the ``return_type``, keyword. The valid choices are ``{"axes", "dict", "both"}``. If the ``by`` argument is ``None``,
+
```

Comment:
I don't think you need this with the table below. IOW would rather see the table, then comment if needed on particular ones?